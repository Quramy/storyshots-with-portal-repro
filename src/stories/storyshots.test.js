import initStoryshots from '@storybook/addon-storyshots';

// Workaround for https://github.com/facebook/react/issues/11565#issuecomment-368877149
const ReactDOM = require('react-dom');
const React = require('React');
ReactDOM.createPortal = node => React.createElement(node, null);

initStoryshots();
