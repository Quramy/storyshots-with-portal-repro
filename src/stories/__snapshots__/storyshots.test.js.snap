// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Portal portal 1`] = `
<div>
  <p>
    Non portal
  </p>
  <portal />
</div>
`;
